cmd/anonymizer/app/uiconv/fixtures/trace_invalid_json.json

```diff
@@ -0,0 +1,3 @@
+[{"traceID":"2be38093ead7a083","spanID":"7606ddfe69932d34","flags":1,"operationName":"a071653098f9250d","references":[{"refType":"CHILD_OF","traceID":"2be38093ead7a083","spanID":"492770a15935810f"}],"startTime":1605223981761425,"duration":267037,"tags":[{"key":"span.kind","type":"string","value":"server"}],"logs":[],"process":{"serviceName":"16af988c443cff37","tags":[]},"warnings":null},
+  {"traceID":"2be38093ead7a083","spanID":"7bd66f09ba90ea3d","flags":1,"operationName":"471418097747d04a","startTime":1605223981965074,"duration": "invalid","tags":[{"key":"span.kind","type":"string","value":"client"}]}
+]
```

Comment:
This is rather hard to grok, could you simplify this file so that the "wrong" part is obvious? I don't think it needs the full data for that.

Reply:
what about the two below
 * cmd/anonymizer/app/uiconv/fixtures/trace_scan_error.json
 * cmd/anonymizer/app/uiconv/fixtures/trace_wrong_format.json

can they also be simplified?

Reply:
Sure.